refinery/ui/source/js/commons/data-sets/search-api.spec.js

```diff
@@ -68,7 +68,7 @@ describe('DataSet.search-api: unit tests', function () {
       'hl.simple.post': '%3C%2Fem%3E',
       'hl.simple.pre': '%3Cem%3E',
       q: _query,
-      qf: 'title%5E0.5+accession+submitter+text',
+      qf: 'title%5E0.5+accession+submitter+text+description',
```

Comment:
@scottx611x You are finding different data sets because of the switch to EdgeNGram not because the information wasn't there. The memory foot print is expected to be the same (there is an overhead on indexing) but the size of the indices might increase. Using EdgeNGram is fine but now you search twice on `description` (you search on the description field alone and in combination with the text field). This might not have a huge impact but could slow things down in the future or artificially boost hits in the description.

I would also test few more search queries to see if side effects pop up (I ran into this problem after I added synonym search for example, but only for some queries). E.g.:
- ES
- iPS
- cel
- RNA
- ...